src/types/schema.ts

```diff
@@ -184,6 +184,9 @@ export interface FormQuestionOptions {
   comment?: string;
   orientation?: 'vertical' | 'horizontal';
   shownCommentOptions?: { validators?: Array<Record<string, any>>; hide?: { hideWhenExpression: string } };
+  attribute?: {
+    type?: string;
+  };
```

Comment:
Looking at the ampath schema [docs for personAtttribute](https://ampath-forms.vercel.app/docs/core-concepts/questions#personattribute), we could eliminate the need for this being an object for backwards compatibility as well have a single format unless you have reasons for going this route

<img width="791" alt="Screenshot 2024-11-29 at 07 10 58" src="https://github.com/user-attachments/assets/9150a0ed-3541-426c-9662-889e8a268776">

Reply:
Let me work on this